multimodal/src/autogluon/multimodal/utils/object_detection.py

```diff
@@ -807,8 +807,7 @@ def cocoeval_torchmetrics(outputs: List):
     """
 
     import torch
-
-    from . import MeanAveragePrecision
+    from torchmetrics.detection.mean_ap import MeanAveragePrecision
```

Comment:
We can do this for now for quickly moving to torch 2.0. And later figure out how to speed up the map in torch 2.0, as current torchmetrics version is very slow in map calculation.

Reply:
fyi there's an open performance PR worth watching: https://github.com/Lightning-AI/torchmetrics/pull/1389

Reply:
^ will probably be superceded, see also the discussion in the linked issue

https://github.com/Lightning-AI/torchmetrics/pull/1832